### DISCUSSION TITLE
Including Block's file in template

### DISCUSSION THREAD

**Discussion by thoanny** [sulu/sulu#6504] [Q&A] [answered]
Hello,

In that guide, it mentions "block definitions should be stored in individual XML files and imported into the XML template" : https://sulu.io/guides/structured-content-with-blocks

but, not documented in https://docs.sulu.io/en/2.4/book/templates.html#content-blocks

Any idea of how it works?
Thanks <3

**Comment by alexander-schranz** [comment, marked as answer]
Hello @thoanny,

You can split an xml into several files using the [`xi:include`](https://de.wikipedia.org/wiki/XInclude) you can find this in the following section in the docs: https://docs.sulu.io/en/2.4/book/templates.html#including-other-templates. Please let us know if we can improve there something. Or if you want to contribute something [here](https://github.com/sulu/sulu-docs/blob/2.3/book/templates.rst#including-other-templates) we are always happy for Pull Request which improve the documentation.

Also have a look at the Demo for some examples: https://github.com/sulu/sulu-demo/blob/bd4c742e6d5e7fb5fdff050bd829e90023431c18/config/templates/pages/default.xml#L92-L93
